README.md

```diff
@@ -8,36 +8,54 @@ A benchmarking utility for high performance computing based on lattice field the
 
 ## Requirements
   
-  Requires an MPI library and compiler.
+Requires an MPI library and compiler.
 
 ## Set Up
```

Comment:
The addition of the "Setup with Spack" heading below has highlighted to me that every heading apart from this one (and "Miscellaneous") is a noun, but this is a verb. For consistency, could you update this to "Setup", please? Everything else looks great; I'll merge once that's done.

Reply:
Just noticed this comment is not where I placed it. I specifically mean changing line 13 here. (I.e. what you wrote was correct, it's the existing text that needed updating.)

Reply:
Thanks for the clarification.
The next commit fixes that.